lib/publiccloud/ssh_interactive.pm

```diff
@@ -144,6 +145,7 @@ sub prepare_ssh_tunnel {
 
     # Permit root passwordless login over SSH
     $instance->ssh_assert_script_run('sudo sed -i "s/PermitRootLogin no/PermitRootLogin prohibit-password/g" /etc/ssh/sshd_config');
+    $instance->ssh_assert_script_run('sudo sed -iE "/^[\#]\?AllowTcpForwarding/c\AllowTcpForwarding yes\" /etc/ssh/sshd_config') if (is_ec2() && is_hardened());
```

Comment:
why we doing this only for *Amazon* hardended images ? is this setting is set by default for Azure and Google but not for Amazon ?

Reply:
True.